cylc/flow/network/schema.py

```diff
@@ -1690,30 +1709,43 @@ class Arguments(TaskMutation.Arguments):
         reflow = Boolean()
 
 
+def _mut_field(cls):
+    """Convert a mutation class into a field.
+
+    Sets the field metadata appropriately.
+
+    Args:
+        field (class):
+            Subclass of graphene.Mutation
+
+    Returns:
+        graphene.Field
+
+    """
+    return cls.Field(description=cls._meta.description)
+
+
 # Mutation declarations
 
 class Mutations(ObjectType):
     # workflow actions
-    broadcast = Broadcast.Field(description=Message._meta.description)
```

Comment:
The reason I added the `_mut_field` thing can be found on this line where the `broadcast` mutation was accidentally given the `message` metadata.